.gitignore

```diff
@@ -1,3 +1,4 @@
 jekyll/**
 public/**
 hugout/**
+*.swp
```

Comment:
Nitpick: I would prefer to exclude editor-specific cruft from gitignore files. A vim user can fix this for their own system with a user-wide gitignore or `.git/info/excludes`.